api/api.go

```diff
@@ -41,6 +41,9 @@ type API struct {
 
 // Version returns the version information of the GatewayD.
 func (a *API) Version(context.Context, *emptypb.Empty) (*v1.VersionResponse, error) {
+
+	RecordRequestMetrics("Version", "/api/version")
```

Comment:
Add version (v1) to all the endpoint labels, for example:
```suggestion
	RecordRequestMetrics("Version", "/api/v1/version")
```